custom_components/pun_sensor/coordinator.py

```diff
@@ -205,12 +204,9 @@ async def update_fascia(self, now=None):
         )
 
         _LOGGER.info(
-            "Nuova fascia corrente: F%s (prossima: F%s)",
+            "Nuova fascia corrente: F%s (prossima: F%s) \nProssimo cambio fascia: %s",
```

Comment:
Insisto con il non creare una riga aggiuntiva (`\n`), ma invece:
```
"Nuova fascia corrente: F%s (prossima: F%s alle %s)",
```
visto che è solo una data/ora in più e si riferisce già alla fascia.

Reply:
ah okok sorry non avevo capito 😅